### PR TITLE
Sync to operator

### PR DESCRIPTION
**Pull request by ostromart**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -165,14 +163,18 @@ require (
 	gopkg.in/yaml.v2 v2.2.4
 	istio.io/api v0.0.0-20191106220915-75ed25ab6842
 	istio.io/gogo-genproto v0.0.0-20190930162913-45029607206a
-	istio.io/operator v0.0.0-20191030215937-82ada6c8f22f
+	istio.io/operator v1.3.0
```
Comment (howardjohn) [inline go.mod]: This doesn't seem right?

Comment (ostromart) [reply go.mod]: not sure what happened there, updated.

**Comment by howardjohn** [comment]
@ostromart the test failure is panic in operator

**Comment by ostromart** [comment]
could you point out the panic? i'm just seeing what seems to be a flake (ratio is a bit off).

**Comment by howardjohn** [comment]
https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio/18777/integ-new-install-k8s-tests_istio/2869

**Comment by howardjohn** [comment]
also I think the update may be covered by https://github.com/istio/istio/pull/18742?

**Comment by ostromart** [comment]
hmmm. that one has me a bit puzzled. clearly some concurrency problem but at first glance but i don't see anything obvious. will take a look tomorrow, since i don't think you need master for the time being.

**Comment by howardjohn** [comment]
@ostromart I think this sync was actually done in another PR, we had a few people all doing the same thing. It passed there and seems to keep passing so I guess its a pretty rare thing?

going to close this just since its already synced

**Pull request closed without merging**